services/search/src/search/routes/search.py

```diff
@@ -53,7 +53,7 @@
 
 ROW_IDX_COLUMN = "__hf_index_id"
 MAX_ROWS = 100
-UNSUPPORTED_FEATURES = [Value("binary"), Audio()]
+UNSUPPORTED_FEATURES = [Value("binary")]
```

Comment:
Will you remove Audio from rows.py?

Reply:
I'm keeping Audio() in the /rows UNSUPPORTED_FEATURES so that it's still unsupported in general, except for arabic_speech_corpus